warp/yul/BuiltinHandler.py

```diff
@@ -412,6 +412,22 @@ def __init__(self, function_args: str):
         self.function_call = "Uint256(exec_env.calldata_size, 0)"
 
 
+class CallDataCopy(BuiltinHandler):
+    def __init__(self, function_args: str):
+        super().__init__(
+            module="evm.calls",
+            function_name="calldatacopy_",
+            function_args=function_args,
+            call_implicits=["range_check_ptr", "exec_env"],
+            used_implicits=("range_check_ptr", "exec_env", "memory_dict", "msize"),
+        )
+        self.function_call = (
+            f"calldatacopy_{{range_check_ptr=range_check_ptr, exec_env=exec_env}}("
+            f"{self.function_args})\n"
+            f"local exec_env : ExecutionEnvironment = exec_env"
```

Comment:
Wouldn't this copy be performed automatically because "exec_env" is in the list of `used_implicits`?